app/facades/websocket/WebSocketController.scala

```diff
@@ -67,7 +80,7 @@ class WebSocketController @Inject() (actorSystem:  ActorSystem,
     val publicKey = keys.encodedPublicKey
     val dsId = Settings.BrokerName + "-" + keys.encodedHashedPublicKey
 
-    val cr = ConnectionRequest(publicKey, true, true, None, "1.1.2", Some(List("json")), true)
+    val cr = ConnectionRequest(publicKey, true, true, None, "1.1.2", Some(List(MSGJSON, MSGPACK)), true)
```

Comment:
Does order matter in that case? if it does should 'MSGPACK' format be in the first place?

Reply:
No matter. I have not read somewhere what order is meaningful here.

Reply:
Thanks for explanations.